DarkFlippers/unleashed-firmware
# Fix SubGHz last used settings

# What's new

- Fix reset "Decode RAW" value, when switch between "Read" and "ReadRAW" in SubGHz
- Add  additional values to "Backlight Time" in "Settings-> LCD and Notification". Now you can choice from seconds up to 30 minutes
- Refactor code and fix names

# Verification 

- Open "SubGHz -> Read -> Config"
- Change "Detect RAW" to "ON"
- Open "SubGHz -> ReadRAW", try to intercept some data
- Open  "SubGHz -> Read" and verify with RAW values and check "Config"

# Checklist (For Reviewer)
